Essentials/src/messages.properties

```diff
@@ -90,6 +90,7 @@ deleteFileError=Could not delete file\: {0}
 deleteHome=\u00a76Home\u00a7c {0} \u00a76has been removed.
 deleteJail=\u00a76Jail\u00a7c {0} \u00a76has been removed.
 deleteWarp=\u00a76Warp\u00a7c {0} \u00a76has been removed.
+deletedKit=\u00a72The\u00a7c {0}\u00a72 kit successfully deleted.
```

Comment:
Rename the key to `deleteKit` and make the message consistent with the other `deleteXYZ` messages.